### PR TITLE
Add icons

### LINKED ISSUE #299: Add icon to student / schedule / task

### PR DESCRIPTION
**Pull request by HollaG**
Fixes #299

### CODE REVIEW

File: docs/TempUG.md [hunk 1]
```diff
@@ -824,21 +824,24 @@ Failure outputs:
 * Input: `filter -before 2022/10/10 -after 2022/01/01`
   * Error: Both the `-before` and `-after` flags are specified. Use only one of `-before`, `-after`, and `-on` in the same command.
 ```
-TODO
+Invalid filter format: You can only use one of -before, -on, -after at a time. 
+Usage: filter -(at least one of unique [-name|subject|before|on|after|remark VALUE]).
```
Comment (elaineshijie) [inline docs/TempUG.md]: remove remark

File: docs/TempUG.md [hunk 2]
```diff
@@ -569,7 +569,7 @@ Please use show lessonIndex before deleting task!
 
 ### Edit Feature
 
-Edits the specified item in the  ___STUDENT list___ or ___SCHEDULE list___ of the application.
+Edits the specified item in the  ___STUDENT list___ or 📅 ___SCHEDULE list___ of the application.
```
Comment (richiehx) [inline docs/TempUG.md]: student list no icon

**Review by richiehx — COMMENTED** [review]
LGTM, some minor nits

**Review by richiehx — APPROVED** [review]
LGTM!

**Pull request merged**